mongodump/mongodump_test.go

```diff
@@ -1154,3 +1154,57 @@ func TestMongoDumpViews(t *testing.T) {
 
 	})
 }
+
+func TestMongoDumpCollectionOutputPath(t *testing.T) {
+	// testtype.SkipUnlessTestType(t, testtype.UnitTestType)
```

Comment:
Forgot to uncomment this when pushing! I'll add it back in if it's the right type of test for this change.